platform/security/expansion/security-expansion-userattr-map/pom.xml

```diff
@@ -14,24 +14,14 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>security-pep-pom</artifactId>
-        <groupId>ddf.security.pep</groupId>
+        <artifactId>security-expansion-pom</artifactId>
+        <groupId>ddf.security.expansion</groupId>
         <version>2.9.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>security-pep-metacardattr-map</artifactId>
-    <name>DDF :: Security :: PEP :: Metacard Attributes Map</name>
```

Comment:
Yeah, this looks really off here, but is correct locally. I blame git since the one above is the same (user -> metacard).